### DISCUSSION TITLE
the content of header-panel-info.html is showing in the header

### DISCUSSION THREAD

**Discussion by mohammedTBB** [breezefront/community#179] [1. Help]
the content of header-panel-info.html is showing in the header.
reproduced on a clean magento 2.4.6-p1 and no third-party extensions.

`<div class="label switcher-label">Support</div> <a href="mailto:sales@example.com"><svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor"> <path d="M2.003 5.884L10 9.882l7.997-3.998A2 2 0 0016 4H4a2 2 0 00-1.997 1.884z" /> <path d="M18 8.118l-8 4-8-4V14a2 2 0 002 2h12a2 2 0 002-2V8.118z" /> </svg> sales@example.com</a> <a href="tel:"><svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor"> <path fill-rule="evenodd" d="M7 2a2 2 0 00-2 2v12a2 2 0 002 2h6a2 2 0 002-2V4a2 2 0 00-2-2H7zm3 14a1 1 0 100-2 1 1 0 000 2z" clip-rule="evenodd" /> </svg> </a>
`
Is there a way to hide it without touching the code?

**Comment by vovayatsyuk** [comment]
Sorry, I don't understand what is wrong, and what are you trying to fix.
